install_pince.sh

```diff
@@ -91,6 +92,10 @@ case "$OS_NAME" in
     INSTALL_COMMAND="-S"
     PIP_COMMAND="pip"
     ;;
+*Fedora*)
+    PKG_MGR="dnf -y"
```

Comment:
Had to add extra command line option to get it to work, this will break dnf if it attempts to perform some other operations but this should not matter in this case